packages/core/src/runtime/Test.ts

```diff
@@ -53,7 +55,8 @@ export default class Test implements ITest {
 
 	public failed: boolean
 
-	public stepCount: number
+	public summaryStep: SummaryStep[] = []
+	public subTitle: string = ''
```

Comment:
eslint should give you the error `Type string trivially inferred from a string literal, remove type annotation` 
```suggestion
	public subTitle = ''
```